src/Bicep.LangServer/SemanticTokenVisitor.cs

```diff
@@ -409,5 +429,17 @@ public override void VisitObjectTypePropertySyntax(ObjectTypePropertySyntax synt
             Visit(syntax.Colon);
             Visit(syntax.Value);
         }
+
+        private static SemanticTokenType GetSemanticTokenForPotentialTypeSymbol(Symbol? symbol) =>
+            symbol switch
+            {
+                PropertySymbol property => GetSemanticTokenForPotentialTypeSymbol(property.Type),
```

Comment:
Will this end up highlighting an imported variable (accessed as a property on a wildcard import) as a type? E.g.,

mod.bicep
```bicep
@export()
var foo = 'foo'
```
main.bicep
```bicep
import * as mod from 'mod.bicep'

output foo string = mod.foo
```

Reply:
It highlights the `mod` part as a namespace but `foo` is highlighted as a variable:
![image](https://github.com/Azure/bicep/assets/22460039/03e9e2d9-ba86-460c-99ae-d670805cf6ec)